### PR TITLE
Simplify audio and segments building

### PR DESCRIPTION
**Pull request by ahmedre**
This patch goes with https://github.com/quran/quran.com-api/pull/73,
which removes oggs and simplifies the response from the server.

### CODE REVIEW

**Comment by ahmedre** [comment]
note that the Jenkins container that builds will likely not work, since it will be built against the current api as opposed to quran/quran.com-api#73.

i've deployed an instance here: http://staging.quran.com:8001

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32800

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32801

File: src/helpers/buildAudio.js [hunk 1]
```diff
@@ -1,69 +1,30 @@
-export function testIfSupported(ayah, agent) {
+export function testIfSupported(ayah) {
```
Comment (mmahalwy) [inline src/helpers/buildAudio.js]: don't need me anymore :)

File: src/redux/modules/audioplayer.js [hunk 2]
```diff
@@ -95,7 +95,6 @@ export default function reducer(state = initialState, action = {}) {
       let currentFile;
       const isSupported = testIfSupported(
```
Comment (mmahalwy) [inline src/redux/modules/audioplayer.js]: don't need this anymore!

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32803

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32804

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32805

**Comment by ahmedre** [comment]
jenkins was building against prod api - i've just changed this, so let me try re-building.

**Comment by ahmedre** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32806

**Comment by ahmedre** [comment]
yep - above build works. note that staging.quran.com is broken at the moment because master frontend is incompatible with master api, which this patch fixes.

File: src/redux/modules/audioplayer.js [hunk 3]
```diff
@@ -93,10 +93,8 @@ export default function reducer(state = initialState, action = {}) {
     case AYAHS_LOAD_SUCCESS: {
       debug('reducer:audioplayer', 'AYAHS_LOAD_SUCCESS init');
       let currentFile;
-      const isSupported = testIfSupported(
-        action.result.entities.ayahs[action.result.result[0]],
-        state.userAgent
-      );
+      const isSupported =
```
Comment (mmahalwy) [inline src/redux/modules/audioplayer.js]: don't need this anymore. and delete line 99's if clause :)

Comment (ahmedre) [inline src/redux/modules/audioplayer.js]: i agree - will do this in a separate patch though
